cli/render.go

```diff
@@ -42,20 +45,26 @@ func orderFull(order exchange.Order) string {
 		r["completed_at"] = order.Completed
 		r["fee"] = order.Fee
 	}
-	str, _ := json.MarshalIndent(r, "", "    ")
+	str, err := json.MarshalIndent(r, "", "    ")
+	if err != nil {
+		panic(err)
+	}
 	return string(str)
 }
 
 func orderbookFull(orderbook exchange.MarketRecord) string {
-	str, _ := json.MarshalIndent(orderbook, "", "    ")
+	str, err := json.MarshalIndent(orderbook, "", "    ")
+	if err != nil {
+		panic(err)
```

Comment:
in general these methods should `return "", err`, but can leave it as for now since this is in the cli tool and we're not sure if we're even using this code